src/styles/main.scss

```diff
@@ -0,0 +1,4 @@
+@import 'utils/variables';
+@import 'blocks/calendar';
+@import 'blocks/page';
+@import 'blocks/reset';
```

Comment:
Mind import order here - probably `reset` should go first

Reply:
Understood thanks